back-end/src/APIs/repositories/commentRepository.ts

```diff
@@ -28,3 +28,7 @@ export const updateCommentRepository = async (id: string, comment: CommentModel)
 export const deleteCommentRepository = async (id: string) => {
 	await knexConnection('comments').where('id', id).del();
 };
+
+export const deleteCommentRepositoryByContent = async (content: string) => {
```

Comment:
This method will also delete every single comment, that contains the content. Contents are non-unique. If using it only for db clean-up after test, it would be nice to have unique naming for it.